### PR TITLE
Implement setIMEKeyboardState function for Tizen

### PR DESCRIPTION
**Pull request by pyrasis**
- Implement setIMEKeyboardState
- Modify space character check.

### CODE REVIEW

File: cocos2dx/platform/tizen/CCImage.cpp [hunk 1]
```diff
@@ -159,7 +159,7 @@ class BitmapDC
 				return false;
 			}
 
-			if (isspace(unicode)) {
+			if (unicode == ' ') {
```
Comment (dumganhar) [inline cocos2dx/platform/tizen/CCImage.cpp:162]: Why to do this change?

**Comment by pyrasis** [comment]
"안녕하세요" is hello in Korean.

isspace function recognizes some unicode Korean characters as a space character.

안 -> space character
녕 -> ok
하 -> ok
세 -> space character
요 -> space character

isspace(unicode)
isspace

unicode == ' '
equalspace

Do you have any idea?

**Comment by dumganhar** [comment]
No ideas about that.

**Comment by dumganhar** [comment]
Why not use `Landscape` mode for HelloCpp ?

**Comment by pyrasis** [comment]
No reason. I will modify to set landscape mode.

**Pull request merged**